okhttp/src/main/java-templates/okhttp3/internal/Version.kt

```diff
@@ -14,6 +14,7 @@
  * limitations under the License.
  */
 @file:JvmName("Version")
```

Comment:
Should we use lowercase file name, e.g., `version.kt`, instead of `Version.kt`?

Reply:
I don't want to change any `internal.*` packages, only internal-visibility in public packages.

Reply:
Ah, just to confirm, we don't change anything for `internal.*` packages(kotlin files), e.g., hostnames.kt and dates.kt as well? Will undo the changes if you meant so :)